src/js/components/purchaseOrder/PurchaseOrderList.jsx

```diff
@@ -24,7 +24,7 @@ const PurchaseOrderList = (props) => {
     props.fetchTranslations(props.locale, 'reactTable');
   }, [props.locale]);
 
-  const isCentralPurchasingEnabled = props.supportedActivities.includes('ENABLE_CENTRAL_PURCHASING');
+  const isCentralPurchasingEnabled = useMemo(() => props.supportedActivities.includes('ENABLE_CENTRAL_PURCHASING'), [props.supportedActivities]);
```

Comment:
I think the using `useMemo` here actually costs more than re-rendering this part, as it is not that complexed. Don't you think so?

Reply:
You are totally right, I thought only about complexity of `includes()` method which is O(n). I forgot that `props.supportedActivities` is not so big to have impact on time of this rerenders.